runtime/v2/shim/util_unix.go

```diff
@@ -25,11 +25,11 @@ import (
 	"net"
```

Comment:
What about `util_windows.go` `AnonDialer`?

Reply:
Erm... what about it? 59374543f4aba28e5ac16ebb1875034ff52c9dab reverts my initial changes and brings back that 5 second timeout as before.

Reply:
So we are not going to use dialer for it then?

Reply:
I don't really see a good reason for it, unless that comment is wrong:

> If there is nobody serving the pipe we limit the timeout for this case to
> 5 seconds because any shim that would serve this endpoint should serve it
> within 5 seconds.

If it's correct, we would still have to create a context with timeout of 5 seconds and wait for it, then call the timeoutDialer which does time.After(timeout) and calls `winio.DialPipe` where yet another timeout is handled with a context.

I see no reason to do that timeout handling on so many layers.

Reply:
@jterry75 does this make sense? Should I anyways use the `pkg/dialer`?

Reply:
Im fine with not merging them I was just making sure you didnt miss it by accident